apps/cookbook/src/app/examples/tabs-example/tab/tab-example.component.ts

```diff
@@ -56,13 +56,13 @@ import { ActionSheetItem } from '@kirbydesign/designsystem';
 
       <kirby-fab-sheet *kirbyPageContent="{ fixed: true }" horizontalAlignment="right">
         <kirby-icon name="write-message"></kirby-icon>
-        <kirby-action-sheet
+        <kirby-action-sheet-popout
```

Comment:
Changed the API to avoid breaking change. It is now possible to optionally instruct the action-sheet to hide the trigger button. FAB and page showcases was updated to reflect this